### PR TITLE
Feat/civ 3246

### PR DESCRIPTION
**Pull request by dtortolaV1**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CIV-3246
https://tools.hmcts.net/jira/browse/CIV-3247

### Change description ###
Added phone and email fields for individual and sole trader applicants

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by kurt-uk — DISMISSED** [review]

**Review by sankaviv1 — APPROVED** [review]

File: ccd-definition/CaseEventToComplexTypes/CreateClaim-HL-nonprod.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    "FieldShowCondition": "applicant1.type = \"INDIVIDUAL\" OR applicant1.type = \"SOLE_TRADER\""
+  },
+  {
+    "ID": "Claimant",
+    "CaseEventID": "CREATE_CLAIM",
+    "CaseFieldID": "applicant2",
+    "ListElementCode": "partyEmail",
+    "FieldDisplayOrder": 20,
+    "DisplayContext": "OPTIONAL",
+    "FieldShowCondition": "applicant1.type = \"INDIVIDUAL\" OR applicant2.type = \"SOLE_TRADER\""
```
Comment (douglasrice) [inline ccd-definition/CaseEventToComplexTypes/CreateClaim-HL-nonprod.json:27]: Should this be referring to applicant 1 when it is the email for applicant2?

File: ccd-definition/CaseEventToComplexTypes/CreateClaim-HL-nonprod.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  },
+  {
+    "ID": "Claimant",
+    "CaseEventID": "CREATE_CLAIM",
+    "CaseFieldID": "applicant2",
+    "ListElementCode": "partyPhone",
+    "FieldDisplayOrder": 21,
+    "DisplayContext": "OPTIONAL",
+    "FieldShowCondition": "applicant1.type = \"INDIVIDUAL\" OR applicant2.type = \"SOLE_TRADER\""
```
Comment (douglasrice) [inline ccd-definition/CaseEventToComplexTypes/CreateClaim-HL-nonprod.json:36]: Should this be referring to applicant 1 when it is the phone for applicant2?

File: ccd-definition/CaseEventToComplexTypes/CreateClaimLRspec-HL-nonprod.json [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    "FieldShowCondition": "applicant1.type = \"INDIVIDUAL\" OR applicant1.type = \"SOLE_TRADER\""
+  },
+  {
+    "ID": "Claimant",
+    "CaseEventID": "CREATE_CLAIM_SPEC",
+    "CaseFieldID": "applicant2",
+    "ListElementCode": "partyEmail",
+    "FieldDisplayOrder": 20,
+    "DisplayContext": "OPTIONAL",
+    "FieldShowCondition": "applicant1.type = \"INDIVIDUAL\" OR applicant2.type = \"SOLE_TRADER\""
```
Comment (douglasrice) [inline ccd-definition/CaseEventToComplexTypes/CreateClaimLRspec-HL-nonprod.json:27]: Should this be referring to applicant 1 when it is the email for applicant2?

File: ccd-definition/CaseEventToComplexTypes/CreateClaimLRspec-HL-nonprod.json [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+  },
+  {
+    "ID": "Claimant",
+    "CaseEventID": "CREATE_CLAIM_SPEC",
+    "CaseFieldID": "applicant2",
+    "ListElementCode": "partyPhone",
+    "FieldDisplayOrder": 21,
+    "DisplayContext": "OPTIONAL",
+    "FieldShowCondition": "applicant1.type = \"INDIVIDUAL\" OR applicant2.type = \"SOLE_TRADER\""
```
Comment (douglasrice) [inline ccd-definition/CaseEventToComplexTypes/CreateClaimLRspec-HL-nonprod.json:36]: Should this be referring to applicant 1 when it is the phone for applicant2?

**Review by HarryH96 — APPROVED** [review]

**Review by sankaviv1 — APPROVED** [review]

**Pull request merged**